src/common/header/components/GitHubButton.tsx

```diff
@@ -0,0 +1,21 @@
+import { IconButton } from '@chakra-ui/button';
+import { AiFillGithub } from 'react-icons/ai';
+
+export function GitHubButton(): JSX.Element {
+  return (
+    <IconButton
+      as="a"
```

Comment:
you can delete this file, it got added and then removed but git thinks you're adding it.